Button and logo added to social media profile links.

adnanraghib1308 opened the pull request:
To make the links more attractive added button and logo of social media in the social media links section.
See the screen shot


![Screenshot 2020-12-26 at 9 25 44 AM](https://user-images.githubusercontent.com/56679676/103145261-5b0c5e80-475d-11eb-90b3-9f5b1bb2bb80.jpg)

adnanraghib1308 commented:
@ankushdharkar please review this

sumitd94 commented on index.html:
```diff
@@ -58,13 +58,36 @@ <h1 class="welcome-title"> Welcome to Real Dev Squad </h1>
         <span>|</span>
-        <a href="https://twitter.com/realdevsquad" target="_blank">Twitter</a>
+        <a class="twitterlink" 
+        href="https://twitter.com/realdevsquad" target="_blank">
```
Please add `rel="noreferrer"`

adnanraghib1308 replied:
@sumitd94 done

ankushdharkar commented on css/main.css:
```diff
@@ -131,11 +132,87 @@ nav .user-greet .user-greet-msg {
+  text-decoration: none;
+}
+
+.facebook-btn{
+  font-size: 16px;
+  border-radius: 10px;
+  padding: .3rem;
+  background-color: white;
+  cursor: pointer;
+}
+
+
+.logo-facebook{
+  vertical-align: middle;
+  width: 25px;
+  height: 25px;
+}
+
+.facebooklink{
+  text-decoration: none;
+}
+
+.instagram-btn{
+  font-size: 16px;
+  border-radius: 10px;
+  padding: .3rem;
+  background-color: white;
+  cursor: pointer;
+}
+
+.logo-instagram{
+  vertical-align: middle;
+  width: 25px;
+  height: 25px;
+}
+
+.instagramlink{
+  text-decoration: none;
+}
+
```
@sumitd94 How does this CSS look? Any issues?

sumitd94 replied:
Mixture of `rem` and `px` ?

ankushdharkar replied:
@sumitd94 No. Look closely again.

swarajpure replied:
@ankushdharkar same and repeated css for Facebook btn, Twitter btn, LinkedIn btn etc.

ankushdharkar reviewed (changes requested):
@sumitd94 Please check the tag and recommend next steps to @adnan1308

ankushdharkar commented:
@adnanraghib1308 Facebook logo looks squished to me. Please fix its ratio.

Also, let's remove the border around the buttons please.

adnanraghib1308 commented:
@ankushdharkar done. Please see the screenshot for required changes.

ankushdharkar commented:
Thank you @adnanraghib1308. Why is the LinkedIn logo not the right color/fill, compared to the style of the other icons?

Harshi7016 commented:
I was just going through the css and observed text-decoration:none repeated 4 times with different social media links can't it be generalized ?
Just a thought and happy to be corrected

Harshi7016 commented:
And same for logo too

adnanraghib1308 commented:
@ankushdharkar @swarajpure @sumitd94 @Harshi7016 Made css more generalized, removed the repeated css for different social media buttons and links. Fixed the linkedin logo (see screenshot). Please let me know for any other improvement or changes. Thanks.
